docker/images/nginx/nginx.conf

```diff
@@ -0,0 +1,34 @@
+worker_processes 1;
+
+events {
+    worker_connections 1024;
+    accept_mutex off;
+}
+
+http {
+    include mime.types;
+    default_type application/octet-stream;
+
+    upstream django {
+        server web:8000;
+    }
+
+    server {
+        listen 80 default;
+        charset utf-8;
+
+        root /app/static;
```

Comment:
I'll have to look up how wheelhouse works. This was simple to set up but I can iterate on it.